iocage_lib/ioc_json.py

```diff
@@ -780,6 +780,10 @@ def check_config(self, conf, default=False):
         if not conf.get('allow_vmm'):
             conf['allow_vmm'] = '0'
 
+        # Version 18 keys
+        if not conf.get('netgraph'):
+            conf['netgraph'] = 'off'
```

Comment:
Bool PR change:
```suggestion
            conf['netgraph'] = 0
```

Reply:
Appreciate you pointing out all the places that the bool PR affects. I was digging into that last night and worried I might have missed one.

As for the code duplication, I had been deliberately avoiding modifying the existing VNET code in the event that you had some disagreement about my approach with netgraph. It sounds like we are on the same page with how to move forward with merging, so I'll go ahead with refactoring to remove as much duplication as is reasonable.